<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1163</width>
    <height>784</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4" stretch="9,1">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0,8">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3" stretch="5,1">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <property name="sizeConstraint">
             <enum>QLayout::SetDefaultConstraint</enum>
            </property>
            <item>
             <widget class="QPushButton" name="button_savepic">
              <property name="text">
               <string>保存图片</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="button_openpic">
              <property name="text">
               <string>打开图片</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="button_show_oripic">
              <property name="text">
               <string>图片切换</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="button_start">
              <property name="text">
               <string>开始转换</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QLabel" name="display_words1">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="font">
               <font>
                <family>Txt</family>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>转换系数</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="input_dif">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBox_blackwhite">
              <property name="layoutDirection">
               <enum>Qt::LeftToRight</enum>
              </property>
              <property name="text">
               <string>黑白</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBox_sizefixed">
              <property name="text">
               <string>适应大小</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="pic_area">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QScrollArea" name="scrollArea">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>984</width>
             <height>626</height>
            </rect>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTextEdit" name="status_area">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1163</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_H">
    <property name="title">
     <string>帮助(&amp;H)</string>
    </property>
    <addaction name="actiona"/>
   </widget>
   <widget class="QMenu" name="menu_T">
    <property name="title">
     <string>换肤(&amp;T)</string>
    </property>
    <addaction name="action_changeto_psblack"/>
    <addaction name="action_changeto_lightgrey"/>
   </widget>
   <addaction name="menu_H"/>
   <addaction name="menu_T"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actiona">
   <property name="text">
    <string>并没有写</string>
   </property>
  </action>
  <action name="action_changeto_psblack">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>psblack</string>
   </property>
  </action>
  <action name="action_changeto_lightgrey">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>lightgrey</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
